justfile

```diff
@@ -68,6 +68,10 @@ publish:
 publish-external tag *args="":
     cd {{justfile_directory()}} && ./scripts/c8y-publish-release.sh {{tag}} {{args}}
 
+# Publish a given github release to Cumulocity (using external urls) but convert an existing draft to a prerelease beforehand
+publish-external-prerelease tag prerelease="true":
+    cd {{justfile_directory()}} && ./scripts/c8y-publish-release.sh {{tag}} {{prerelease}}
+
```

Comment:
If using a `--pre-release` flag, then this task would change to...the task is includes "prerelease" in the task name, so it doesn't make sense to have a flag to set if it is a pre-release or not...

```suggestion
# Publish a given github release to Cumulocity (using external urls) but convert an existing draft to a prerelease beforehand
publish-external-prerelease tag:
    cd {{justfile_directory()}} && ./scripts/c8y-publish-release.sh {{tag}} --pre-release

```